packages/gatsby-theme-docs/bin/create-release-note

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const slugify = require('slugify');
+const prompts = require('prompts');
+
+const questions = [
+  {
+    type: 'multiselect',
+    name: 'type',
+    message: 'Pick the type of the release note',
+    choices: [
+      { title: 'enhancement', value: 'enhancement' },
+      { title: 'feature', value: 'feature' },
+      { title: 'fix', value: 'fix' },
+      { title: 'announcement', value: 'announcement' },
```

Comment:
It would be nice to add a short explanation of what/how these types are used for.

Reply:
there is a wiki page with the usage instructions in the "let's write the docs" section.

Reply:
Yes I meant to add this information in the prompt, so that people don't have to look it up from the wiki

Reply:
I added the explanation texts from the wiki page to the descriptions of the types now